### DISCUSSION TITLE
调用langchain的openai_embeddings显示参数非法

### DISCUSSION THREAD

**Discussion by DingGao1124** [zai-org/ChatGLM3#1214] [Q&A] [answered]
![731715532214_ pic](https://github.com/THUDM/ChatGLM3/assets/136151131/21d90aa7-a7aa-4162-bbb1-11cbedb03a74)
![721715532195_ pic](https://github.com/THUDM/ChatGLM3/assets/136151131/083ebc7d-22db-470e-85d0-6597763c8b00)
通过langchain的openai_embeddings调用chatglm4的嵌入模型显示prompt非法，是当前还不支持更换api_base吗？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
ebedd没有适配
